common/src/cellular_3gpp_api.c

```diff
@@ -1805,7 +1798,9 @@ CellularError_t Cellular_CommonGetServiceStatus( CellularHandle_t cellularHandle
 {
     CellularContext_t * pContext = ( CellularContext_t * ) cellularHandle;
     CellularError_t cellularStatus = CELLULAR_SUCCESS;
-    cellularOperatorInfo_t operatorInfo = { 0 };
+    cellularOperatorInfo_t operatorInfo;
```

Comment:
Can you describe more about this modification?

Reply:
This one violates rule_10_3_violation:	Implicit conversion of "0" from essential type "signed 8-bit int" to different or narrower essential type "character".

Reply:
After discussion, cellularOperatorInfo_t contains char array.
There will be misra violation if we use "{ 0 }".
This change use memset to initialize the structure. The local variable is initialized before using it.